src/Akeneo/Tool/Bundle/BatchBundle/Job/JobInstanceRepository.php

```diff
@@ -29,4 +30,14 @@ public function getIdentifierProperties()
     {
         return ['code'];
     }
+
+    public function remove(string $identifier): void
+    {
+        $sql = <<<SQL
+    DELETE FROM akeneo_batch_job_instance
+    WHERE code = :code
+SQL;
```

Comment:
```suggestion
        $sql = <<<SQL
            DELETE FROM akeneo_batch_job_instance
            WHERE code = :code
        SQL;
```
We can add indentation in heredoc since PHP 7.4